packages/bits/src/lib/toolbar/toolbar-keyboard.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from "@angular/core";
+import { KEYBOARD_CODE } from "../../constants";
+import { MenuComponent } from "../menu";
+
+@Injectable()
+export class ToolbarKeyboardService {
+    public dynamicContainer: HTMLElement | null;
+
+    public moreBtn: HTMLButtonElement | null;
+
+    public menuComponent: MenuComponent | null = null;
+
+    public initService(container: HTMLElement, menu: MenuComponent | null): void {
+        this.dynamicContainer = container.querySelector(".nui-toolbar-content__dynamic");
+
+        if (this.dynamicContainer) {
+            this.moreBtn = this.dynamicContainer.querySelector(".nui-button.btn.menu-button");
+        }
+
+        if (menu) {
+            this.menuComponent = menu;
+        }
+    }
+
+    public disableFocusForMoreBtn(): void {
+        if (this.moreBtn) {
+            this.moreBtn.setAttribute("tabindex", "-1");
+        }
+    }
+
+    public onKeyDown(event: KeyboardEvent): void {
+        const { code } = event;
+
+        if (code === KEYBOARD_CODE.ARROW_LEFT || code === KEYBOARD_CODE.ARROW_RIGHT) {
+            event.preventDefault();
+            this.navigateByArrow(code);
+        }
+    }
+
+    private navigateByArrow(code: string): void {
```

Comment:
Can we used the KEYBOARD_CODE as the type instead of string?

Reply:
When using the CDK constants mentioned above, it would be just numbers.